charts/opentelemetry-ebpf/ci/enabled-log-debug-values.yaml

```diff
@@ -10,4 +10,4 @@ log:
 debug:
   enabled: true
   storeMinidump: true
-  sendUnplannedExitMetric: false
```

Comment:
I think we should use same idea I mentioned on #1347 
let's make it a different test file